themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -136,7 +136,7 @@
       <?php if (isset($this->layout()->srmessage)): // message for benefit of screen-reader users ?>
         <span class="sr-only" role="heading" aria-level="1"><?=$this->layout()->srmessage ?></span>
       <?php endif; ?>
-      <a class="sr-only" href="#content"><?=$this->transEsc('Skip to content') ?></a>
+      <a class="sr-only-focusable" href="#content"><?=$this->transEsc('Skip to content') ?></a>
```

Comment:
I can see the value in making this focusable for the benefit of keyboard users. However, I seem to recall @EreMaijala may have said something about the sr-only class in relation to the ongoing work to upgrade to Bootstrap 5, so I'm just flagging him here in case he has any concerns about this. (It's entirely possible that I'm misremembering something, and there's no problem here).

Reply:
In bootstrap 5 `.sr-only` and `.sr-only-focusable` have been renamed to `.visually-hidden` and `.visually-hidden-focusable`.